### PR TITLE
Dashboard tests

### LINKED ISSUE #15: Add tests for dashboard creation
- Test with no feedstocks
- Test with one feedstock
- Test with different filename
- Test number of rows matches expected

### PR DESCRIPTION
**Pull request by thomashopkins32**
Created tests for the dashboard as described in #15 

Also changed the alternate text for badges to be more descriptive.

Closes #15

### CODE REVIEW

**Comment by thomashopkins32** [comment]
Need to add `.netrc` config for GitHub to have our TravisCI builds pass. Right now it is trying to get all the feedstock repo names from GitHub without authentication. This is the default behavior for creating the dashboard.

**Comment by mrakitin** [comment]
> Need to add `.netrc` config for GitHub to have our TravisCI builds pass or change the way we authenticate. Right now it is trying to get all the feedstock repo names from GitHub without authentication. This is the default behavior for creating the dashboard.

It's easy to set up. Please follow this approach from our other project: https://github.com/srwpy/srwpy/blob/a54c1f3a4f95e15ff841aff3603474daf2b7798c/.travis.yml#L114-L116 (the predefined, secure, variables should be defined first).

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ coverage
 flake8
 pytest
 sphinx
+markdown
+bs4
+lxml
```
Comment (mrakitin) [inline requirements-dev.txt:10]: A sorted or grouped & sorted list would be great.

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # These are required for developing the package (running the tests, building
 # the documentation) but not necessarily required for _using_ it.
+beautifulsoup4
```
Comment (mrakitin) [inline requirements-dev.txt:3]: Good catch about the name of the package!

**Pull request merged**